grow_cluster_test.py

```diff
@@ -183,5 +183,29 @@ def test_grow_3_to_4_large_partition(self):
         self.grow_cluster(cluster_target_size=4,
                           stress_cmd=self.get_stress_cmd_profile())
 
+    def get_benchmark_stress_cmd(self):
+        """
+        Get a cassandra stress cmd string suitable for bencharmking grow cluster purposes.
+
+        :return: Cassandra stress string
+        :rtype: basestring
+        """
+        ip = self.db_cluster.get_node_private_ips()[0]
+        return ("cassandra-stress write cl=QUORUM duration=24000m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(10240) n=FIXED(1)' -pop seq=1..10000000 -node %s" % ip)
+
+    def test_cassandra_growth_benchmark(self):
+        """
+        Benchmark cluster growth
+        """
+
+        ebs_ids = ["vol-048b3e724b7f6990f", "vol-08c963ebdf37b2e1c", "vol-024b835dd076a6d04"]
```

Comment:
can we extract this info into the yaml so we can reuse it in other tests as well